plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
+  ## default is 1. setting to 0 to force disconnect after use.
+  max_idle = 1
+
+  ## maxopen - specify the maximum number of connections to maintain. see sql.DB for details.
+  ## default is 1.
+  max_open = 1
```

Comment:
This setting doesn't currently serve a purpose. Both the `postgresql` and `postgresql_extensible` plugins only execute one query at a time. Thus they'll never open more than 1 connection.